modular_skyrat/master_files/code/modules/clothing/under/syndicate.dm

```diff
@@ -5,13 +5,11 @@
 /obj/item/clothing/under/syndicate
 	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/syndicate_digi.dmi' // Anything that was in the syndicate.dmi, should be in the syndicate_digi.dmi
 
-
 /obj/item/clothing/under/syndicate/skyrat
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/syndicate.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/syndicate.dmi'
 	//These are pre-set for ease and reference, as syndie under items SHOULDNT have sensors and should have similar stats; also its better to start with adjust = false
 	has_sensor = NO_SENSORS
-	armor_type = /datum/armor/clothing_under/syndicate_skyrat
```

Comment:
Why did this have to be removed, anyway?

Reply:
it was the same values as the base tactical turtleneck and i thought it should just inherit from the parent. if you want i guess i could put it back but i don't really see the value in doing that

Reply:
Fair.